addons/mail/tests/test_mail_user_settings.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.tests.common import users
+
+class TestMailUserSettings(MailCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestMailUserSettings, cls).setUpClass()
```

Comment:
just super().setUpClass() works too, but in this case it is a useless override no?

Reply:
removed